assets/app/scripts/controllers/pod.js

```diff
@@ -26,6 +26,12 @@ angular.module('openshiftConsole')
       }
     ];
 
+    // Check for a ?tab=<name> query param to allow linking directly to a tab.
+    if ($routeParams.tab) {
+      $scope.selectedTab = {};
```

Comment:
should this be selectedTab = $routeParams.tab since you can only ever have one tab selected?  any reason to have it as a hash?